File: round 1/Stephenites/problemtest.java
import java.util.*;
public class problem1
{
public static void main(String args[])
{
Scanner in=new Scanner(System.in);
int n,i;
System.out.println("Enter a Number:");
n=in.nextInt();
for(i=0;i<n;i++)
{
if(i%2==0)
{
System.out.println("Yes it is a Prime Number");
}
else
{
System.out.println("No it is not a Prime Number");
}
}
}
}

File: q/c/submissions/goldensnitch/problemtest.c
#include<stdio.h>
int main()
{
   int a,i,f=0;
   printf("Enter the number \n");
   scanf("%d",&a);
   for(i=2;i<=a-1;i++)
    {  
     if(a%i==0)
       f++;
     }
     if(f==0)
    printf("%d is prime",a);
     else
    printf("%d is not prime",a);
    return 0;
}

